AventusClassicJS/Web3ContractsManager.js

```diff
@@ -30,8 +29,9 @@ class Web3ContractsManager {
 
   async getStorage() {
     if (!this.aventusStorage) {
-      const storageDescriptor = await common.getStorageContractDescriptor(this.networkType);
-      this.storageAddress = (this.networkType === 'main') ? MAINNET_STORAGE_ADDRESS : storageDescriptor.address;
+      const networkType = await web3.eth.net.getNetworkType()
+      const storageDescriptor = await common.getStorageContractDescriptor(networkType);
```

Comment:
have we updated `getStorageContractDescriptor` to handle network types returned by web3?

Reply:
they all use getStorageFileName, which happens to work with either network name OR network type... it's not ideal, though :-/ We have the same mess in the main protocol